pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[project]
+name = "xcp-ng-build"
```

Comment:
it should either match the repository name or the command name IMO, ie either `xcp-ng-build-env` or `xcp-ng-dev` (or any other name we'll finally choose)